x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/function/scalar/whitelist/InternalSqlScriptUtils.java

```diff
@@ -113,6 +115,10 @@ public static Boolean notNull(Object expression) {
         return IsNotNullProcessor.apply(expression);
     }
 
+    public static Boolean in(Object value, List<Object> values) {
+        return In.doFold(value, values);
```

Comment:
The naming convention is to call the method on the processor (typically `InProcessor.apply`)